### DISCUSSION TITLE
Recursive forecast using linear regression, single time series, and with lag & future-known features

### DISCUSSION THREAD

**Discussion by AhmedThahir** [alan-turing-institute/sktime#5358] [Q&A] [answered]
I am aware of how to achieve the same in Darts, but I couldn't find a like-to-like way to achieve the same in sktime.

[I need the equivalent of this](https://github.com/sktime/sktime/files/12802092/darts.pdf)

Kindly help me out with this.

Please note that I'm very new to the world of forecasting so excuse me if I missed out on something, so if it's already available in the documentation, kindly point me out to the correct references.

**Comment by yarnabrina** [comment]
Please take a look at [`make_reduction`](https://www.sktime.net/en/stable/api_reference/auto_generated/sktime.forecasting.compose.make_reduction.html).

**Comment by AhmedThahir** [comment]
Hiiii, I did check on make_reduction, but I wasn't able to understand how to include future known covariates.

**Reply by fkiraly** [reply, marked as answer]
You include them as `X` in `predict` (or `predict_interval` etc)

**Reply by AhmedThahir** [reply]
Thanksss!
